### ISSUE TITLE
bibchk not working

### ISSUE THREAD

**Issue by ghost** [BibTheque/bibchk#1]
I just installed it without problems.

But it seems not to recognize given DOI values.

**Comment by dkllrjr** [comment]
Yeah it's currently broken and I haven't had time to fix it. I might be able to do that relatively quickly. I will refactor the code a little.
